.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
+    <bytecodeTargetLevel target="1.8" />
```

Comment:
@tmicro do you know why this is changing?

Reply:
It's probably because Gavin has a different version of Android Studio.  Let's not merge this change.  Let's add this file to .gitignore